packages/retry/remove-manpage.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/Makefile.am b/Makefile.am
+index 67144fc..5ec0dc5 100644
+--- a/Makefile.am
++++ b/Makefile.am
+@@ -5,8 +5,3 @@ bin_PROGRAMS = retry
+ retry_SOURCES = retry.c
+ 
+ EXTRA_DIST = retry.spec
+-dist_man_MANS = retry.1
+-
+-retry.1: retry.c $(top_srcdir)/configure.ac
+-	which txt2man && ./retry --help | txt2man -d 1 -t "${PACKAGE_NAME}" -r "${PACKAGE_NAME}-${PACKAGE_VERSION}" > retry.1 || true
```

Comment:
I guess man page generate was removed because we did not have txt2man?

I've added it to the docker image in https://github.com/termux/termux-packages/commit/8b2b5cc28d491e5a10148ad3c0761cb871c3ccf1, so please remove the patch, thanks!

Reply:
The manpage is the same as `--help` switch. And it can't be generated on host while building. It needs to be done with the `termux_step_create_debscripts` hook. I decided against this. However if you want, I can just do so.